web/src/components/EventsDateFilter.tsx

```diff
@@ -22,6 +22,9 @@ const DateSection = styled.div`
     align-items: center;
   }
 `
+const Text = styled.span`
+  color: ${props => (props.theme.isContrastTheme ? props.theme.colors.textColor : props.theme.colors.backgroundColor)};
+`
```

Comment:
Its white on light mode.
```suggestion
const Text = styled.span`
color: ${props => props.theme.colors.textColor};`
```